### DISCUSSION TITLE
Feature Crews

### DISCUSSION THREAD

**Discussion by bartr** [retaildevcrews/ngsa#549] [FAQ]
Feature Crews

- Monitoring FC - Ryan / Jim (lead) - Chris, Jon, Nsikan, Kushal, Alfredo (PTO)
- PnP-Deploy FC - Deanna / AK (lead) - Chenmin, Gerardo, Kevin, Natasha, Jafar (50%), Kanio(50%)
- WCNP FC - Deanna / Joseph (lead) - Joaquin, Siva, Anne (PTO)

**Comment by dsturgell** [comment]
PnP Feature Crew:

AK, Kevin, Gerardo, Natasha, Chenmin, Jafar (50%), Kanio(50%), Deanna

**Comment by dsturgell** [comment]
Monitoring Feature Crew: 

Ryan, Jim, Nsikan, Jon, Chris, Kushal

**Comment by dsturgell** [comment]
WCNP Feature Crew:

Joseph, Deanna, Joaquin, Siva, Anne (PTO)
